### PR TITLE
feat: add auto-login doc

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

File: docs/user/player/advanced/auto-login.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Auto Login
+
+:::warning
+
+This feature requires a good understanding of Graasp and might be for the more technically inclined.
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:5]:
```suggestion
This feature requires a good understanding of Graasp and might be for users that are more technically inclined.
```

File: docs/user/player/advanced/auto-login.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+:::
+
+In this article we will use the term `itemLogin` to designate the possibility to log a user with a pseudonym.
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:9]:
```suggestion
In this article we will use the term `itemLogin` to designate the possibility to log a user only with a pseudonym.
```
Or should we have a page about "itemlogin" to reference to? I think it's more "login without an email" than "login with a username"

File: docs/user/player/advanced/auto-login.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Purpose {#purpose}
+
+This feature allows to force the itemLogin account used for a lesson by providing a url that contains all necessary information to the user.
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:13]:
```suggestion
This feature allows to force which itemLogin account will be used for a lesson by providing a url that contains all necessary information for the user.
```

File: docs/user/player/advanced/auto-login.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+##  Use cases {#use-cases}
+
+Potential use-cases for this feature include:
+
+- Studies (like Prolific) where you can provide a personalized link to each participant and you **need** each participant to use a very specific username (for data analysis purposes for example)
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:19]: "you need" but "maybe for example" for analysis purpose. Do you need it or not? Maybe "you want" fits better.

File: docs/user/player/advanced/auto-login.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+## Setting up your lesson {#setup}
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:21]: You use "lesson" but your use-case is a "study". Should we be more generic? Capsule? Activity?

File: docs/user/player/advanced/auto-login.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+In this section we will go over the process for setting up auto login.
+
+1. Create an item
+2. Go to this item sharing page
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:26]:
```suggestion
2. Go to this item's sharing page
```

File: docs/user/player/advanced/auto-login.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+3. In the sharing page enable "Pseudonymized" access with "Pseudonym"
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:27]: Have a warning that it does not work with password?

File: docs/user/player/advanced/auto-login.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+   ![allow access for pseudonymised users](./2024-10-23-pseudo-access.png)
+
+4. Create the link to share with users
+    1. Go to the item page in the player, it should look something like this:
```
Comment (pyphilia) [inline docs/user/player/advanced/auto-login.md:32]:
```suggestion
    1. Go to the item page in the player, it should look like this:
```

**Review by pyphilia — CHANGES_REQUESTED** [review]
It is much more understandable, gg!

**Comment by spaenleh** [comment]
Thank you for your comments @pyphilia !

**Review by pyphilia — APPROVED** [review]

**Pull request merged**